SegmSubmodules/Evalutation.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import json
+import matplotlib.pyplot as plt
+from sklearn.metrics import log_loss, f1_score, r2_score, confusion_matrix, classification_report, roc_curve, auc, \
+    roc_auc_score, precision_score, recall_score
+import keras.backend as K
+
+
+def draw_mask_plots(prediction, ground_truth, plot_file):
+    """Draws plots of the predicted and ground truth mask and saves it to plot_file as image
+
+    Args:
+        prediction: the prediction binary mask vector
+        ground_truth: the ground truth binary mask vector
+        plot_file: path to file where plot will be saved"""
+    # Have to add captions, tags and labels
+    plt.figure()
+    plt.plot([x for x in range(len(prediction))], prediction, [x for x in range(len(prediction))], ground_truth)
+    plt.savefig(plot_file)
+
+
+def count_metrics_on_sample(prediction, ground_truth, json_file):
```

Comment:
`metrics_list` parameter can be added. if empty, use a predefined list of sklearn metrics.